src/components/NanoContract/RegisterNewNanoContractButton.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Missing Hathor header

I think it's missing on all components, can you please review?

Reply:
Done